src/pages/admin/donate/DonatePageAdmin.tsx

```diff
@@ -0,0 +1,5 @@
+import { DonatePageContent } from './components/donate-page-content/DonatePageContent';
+
+export const DonatePageAdmin = () => {
+    return <DonatePageContent></DonatePageContent>;
```

Comment:
could this be shortened?
```suggestion
    return <DonatePageContent />;
```